frontend/src/views/Seedlot/SeedlotCreatedFeedback/index.tsx

```diff
@@ -11,7 +11,7 @@ import './styles.scss';
 
 const SeedlotCreatedFeedback = () => {
   const navigate = useNavigate();
-  const { seedlot } = useParams();
+  const seedlotNumber = useParams().seedlot;
```

Comment:
Why the change here?

Reply:
felt like calling it `seedlot` is not descriptive enough and it looks weird to me in the code 👀. 
For example:
```navigate(`/seedlot/details/${seedlot`)}```
    vs
```navigate(`/seedlot/details/${seedlotNumber}`)}```

 `Your A class seedlot {seedlot}`
    vs
 `Your A class seedlot {seedlotNumber}`

I can change it back if you prefer it the other way

Reply:
Oh, I see, that does makes the it more descriptive. Let's leave like this then.

Reply:
alright my latest commit should have fixed all the issues above